### PR TITLE
Adding redux

### PR DESCRIPTION
**Pull request by Dzastinche**
there is redux store update with 3 reducers,
1 User store,
2 favorite movies store,
3 all movies,
Here is clickup task link : https://app.clickup.com/t/269a0nh

### CODE REVIEW

File: src/redux/redux-reducers/favorites.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import {createSlice} from '@reduxjs/toolkit';
+//reducer for favorite movies
+const favorites = createSlice({
+  name: 'favorite',
```
Comment (zlayabekrija) [inline src/redux/redux-reducers/favorites.jsx]: plural please -> favorites

File: src/redux/redux-reducers/favorites.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  initialState: [],
+  reducers: {
+    addFavMovies(state, action) {2
```
Comment (zlayabekrija) [inline src/redux/redux-reducers/favorites.jsx]: What's this number 2, and what we are returning.
Also, let's try too keep names, more meaningful for example addFavorites

File: src/redux/redux-reducers/movies.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import {createSlice} from '@reduxjs/toolkit';
+//reducer for all movies
+const movies = createSlice({
+  name: 'movies',
+  initialState: [],
+  reducers: {
+    addMovies(state, action) {
```
Comment (zlayabekrija) [inline src/redux/redux-reducers/movies.jsx:7]: Same here, let's return some state.

File: src/redux/redux-reducers/userstore.jsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import {createSlice} from '@reduxjs/toolkit';
+//reducer for the user
+const userStore = createSlice({
+  name: 'userStore',
+  initialState: {email:''},
+  reducers: {
+  },
+});
+export const {addMovies} = movies.actions;
```
Comment (zlayabekrija) [inline src/redux/redux-reducers/userstore.jsx]: Why do we have the  movies slice under the user?

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
Greate work, some minor things to adress. Please work on them, and resubmit review request when ready.

Happy coding.

Also please do not remove points from the template, rather leave them as is.
Please have a look [here](https://github.com/Magnificent-12/movie-app/pull/20)

File: src/redux/redux-reducers/userstore.jsx (hunk 4 above)
Comment (Dzastinche) [reply src/redux/redux-reducers/userstore.jsx]: sry for this mistakes, please check now if i fixed them

**Review by zlayabekrija — APPROVED** [review]
Great work :)

**Review by AmerMekic — APPROVED** [review]

**Review by majraoruc — APPROVED** [review]

**Pull request merged**